### PR TITLE
Lw7 - Day 5

### PR DESCRIPTION
**Pull request by fsansalvadore**
## What kind of change does this PR introduce?

Launch Week 7 - Day 5 release.
https://zone-www-dot-com-git-lw7-day-5-supabase.vercel.app/launch-week

Screenshot 2023-04-14 at 15 22 07

Screenshot 2023-04-14 at 15 22 15

Screenshot 2023-04-14 at 15 23 28

### CODE REVIEW

File: apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+thumb: launch-week-7/day-5-community-highlights/day-5-community-highlights-thumb.jpg
+---
+
+Supabase is a collaborative company. We work with, sponsor, and support as many open source tools as possible. Here’s a few of this community highlights from the past 3 months.
+
+## Ecosystem Partners
+
+![Ecosystem Partners](/images/blog/launch-week-7/day-5-community-highlights/day-5-community-highlights-partners.png)
+
+Our [Integrations Marketplace](/docs/guides/integrations) continues to grow. This past few months has brought a number of new partners and collaborations:
```
Comment (RamiroND) [inline apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx:19]: Shouldn't this link to: https://supabase.com/partners/integrations?

File: apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+### AWS RDS
+
+We’ve worked with the RDS team to build out the pg_tle extension and tooling and integrating it into Supabase. These changes are a stepping stone towards running the Supabase stack on AWS RDS. Special shoutout to [@jkatz](https://github.com/jkatz), who was an instrumental collaborator and the rest of the RDS team who have been helping to transition [PL/Rust](https://github.com/tcdi/plrust) into a Trusted Language.
+
+[Read the press announcement on Amazon.com.](https://aws.amazon.com/blogs/opensource/supabase-makes-extensions-easier-for-developers-with-trusted-language-extensions-for-postgresql/)
```
Comment (RamiroND) [inline apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx:25]: Link is 404

File: apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+[Read about pg_tle at Supabase and our collaboration with AWS.](https://www.notion.so/6f1bac83308846488ee7b01ae5c20200)
```
Comment (RamiroND) [inline apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx:27]: Links to internal notion doc.

File: apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx (hunk 2 above)
Comment (MildTomato) [reply apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx:25]: know what the link should be ?

File: apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx (hunk 1 above)
Comment (MildTomato) [inline apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx:19]:
```suggestion
Our [Integrations Marketplace](/partners/integrations) continues to grow. This past few months has brought a number of new partners and collaborations:
```

File: apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx (hunk 3 above)
Comment (MildTomato) [reply apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx:27]: anyone know what the link should be?

File: apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+
+### OrioleDB Patches
+
+We continue to support the development of [OrioleDB](https://github.com/orioledb/orioledb), who are building scalable storage mechanisms for Postgres using table access methods. The OrioleDB extension is now version [Alpha12](https://github.com/orioledb/orioledb/releases/tag/alpha12) and compatible with Postgres 15.
+
+This release includes various improvements and fixes, including parallel sequential scan of OrioleDB relations. This allows multi-CPU systems to speed up select queries that involve sequential scans. It’s an analogue of heap parallel scan in Postgres core but for index-organized tables. With this base, we’re now implementing methods to build Orioledb indexes using many parallel workers.
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx:177]: shush you

File: apps/www/components/LaunchWeek/lw7_days.ts [hunk 5]
```diff
@@ -238,16 +277,219 @@ const days: WeekDayProps[] = [
...
+                img: images['05-community-01-002'],
+              },
+            ],
+            steps: [],
+          },
+        ],
+      },
+      {
+        title: 'Studio Updates',
+        blog: '/blog/studio-v2',
```
Comment (fsansalvadore) [inline apps/www/components/LaunchWeek/lw7_days.ts:312]: @saltcod what's the final slug going to be for the studio blog post?

File: apps/www/components/LaunchWeek/lw7_days.ts [hunk 6]
```diff
@@ -238,16 +277,219 @@ const days: WeekDayProps[] = [
...
+        thumb: images['05-studio-01-thumb'],
+        steps: [
+          {
+            title: '⌘K',
+            badge: 'New',
+            className: 'span-full',
+            url: '#command-k',
```
Comment (fsansalvadore) [inline apps/www/components/LaunchWeek/lw7_days.ts:319]: @saltcod and we're linking to each blog post section from each card, can you check these?

File: apps/www/components/LaunchWeek/lw7_days.ts [hunk 7]
```diff
@@ -238,16 +277,219 @@ const days: WeekDayProps[] = [
...
+        ],
+      },
+      {
+        title: 'One more thing',
+        hideInBlog: true,
+        steps: [
+          {
+            title: 'dbdev',
+            badge: 'New',
+            blog: '/blog/db-dev',
```
Comment (fsansalvadore) [inline apps/www/components/LaunchWeek/lw7_days.ts:459]: final slug?

File: apps/www/components/LaunchWeek/lw7_days.ts [hunk 8]
```diff
@@ -238,16 +277,219 @@ const days: WeekDayProps[] = [
...
+                mobileImg: images['05-omt-01-002-mobile'],
+              },
+            ],
+            steps: [],
+          },
+          {
+            title: 'Postgres TLE',
+            badge: 'New',
+            thumb: images['05-omt-02-thumb'],
+            blog: '/blog/postgres-tle',
```
Comment (fsansalvadore) [inline apps/www/components/LaunchWeek/lw7_days.ts:477]: final slug?

File: apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx [hunk 9]
```diff
@@ -0,0 +1,198 @@
...
+
+We also want to highlight all the humans who have made meaningful contributions to the Supabase ecosystem since our last Launch Week:
+
+- [Gary](https://github.com/GaryAustin1) and [Olyno](https://github.com/olyno) - official moderators of our [Discord](https://discord.supabase.com) and overall incredibly helpful people. The community would not be the same without them
+- [Bruno D'Luka](https://github.com/bdlukaa) - Caring for the details of Flutter SDK so that we can provide the best DX for Flutter devs
+- [CodewithGuillaume](https://twitter.com/blackevilgoblin) - Creating a series of YouTube videos explaining different components of Supabase.
+- [Daniel Mossaband](https://github.com/DanMossa) - Worked on fixing bugs and adding features to bring the mobile auth to Supabase
+- [David Plugge](https://github.com/david-plugge) - Working on improving the [auth-helpers](https://github.com/supabase/auth-helpers) and fixing issues users report.
+- [Jonathan Wilke](https://twitter.com/jonathan_wilke) - built [SupaStarter.dev](https://supastarter.dev/). A SaaS for building SaaSes.
+- [Maail](https://github.com/maail) - Built the Swift libraries and documentation.
```
Comment (thorwebdev) [inline apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx:191]:
```suggestion
- [grsouza](https://github.com/grsouza) & [Maail](https://github.com/maail) - Built the Swift libraries and documentation.
```

File: apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx [hunk 10]
```diff
@@ -0,0 +1,198 @@
...
+
+It’s becoming easier to build applications with any language.
+
+### Support for Swift, Python, and C#
+
+In the past few months, the community has rallied and shipped 3 new client libraries, complete with Docs:
+
+- Swift:
+  - [Documentation](/docs/reference/swift/introduction) | [Source Code](https://github.com/supabase-community/supabase-swift)
+  - Shout-out to: [@maail](https://github.com/maail)
```
Comment (thorwebdev) [inline apps/www/_blog/2023-04-14-launch-week-7-community-highlights.mdx:137]:
```suggestion
  - Shout-out to: [grsouza](https://github.com/grsouza) & [@maail](https://github.com/maail)
```

**Review by MildTomato — APPROVED** [review]

File: apps/www/_blog/2023-04-14-supabase-studio-2.0.mdx [hunk 11]
```diff
@@ -0,0 +1,163 @@
...
+    [**Table/View definitions**](#tableview-definitions): View SQL definitions for tables and views
+  </li>
+  <li>[**API Autodocs**](#api-autodocs): View auto-generated docs right from the Table Editor</li>
+  <li>
+    [**Support for many tables**](#support-for-many-tables): Table Editor now supports 1000s of
+    tables
+  </li>
+  <li>[**JSON editing**](#json-editing): Improved JSON editing</li>
+  <li>[**Nullable columns**](#nullable-columns): Allow text/bool cells to be null or empty</li>
+</ul>
```
[the remark this answers is not in the document]
Comment (kiwicopple) [reply apps/www/_blog/2023-04-14-supabase-studio-2.0.mdx:50]: can be done after the merge

**Pull request merged**